engine/src/main/java/org/hibernate/search/indexes/impl/IndexManagerHolder.java

```diff
@@ -237,15 +253,9 @@ private IndexManager createIndexManager(String indexName,
 		IndexManagerFactory indexManagerFactory = serviceManager.requestService( IndexManagerFactory.class );
 
 		// create IndexManager instance via the index manager factory
-		String indexManagerImplementationName = properties.getProperty( Environment.INDEX_MANAGER_IMPL_NAME );
 		final IndexManager manager;
 		try {
-			if ( StringHelper.isEmpty( indexManagerImplementationName ) ) {
```

Comment:
How comes these bits have changed here? It's not fully clear to me yet whether that's strictly needed for the actual issue.